client/extensions/woocommerce/app/settings/email/mailchimp/setup-mailchimp.js

```diff
@@ -0,0 +1,315 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import { pick, some, isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { localize } from 'i18n-calypso';
```

Comment:
This is technically an external dependency 😆  but not a big deal - I see it listed in both places often.